packages/wonder-stuff-core/src/__tests__/clone.test.ts

```diff
@@ -37,7 +36,7 @@ describe("#clone", () => {
         // Arrange
         const originalValue = {
             array: [1, 2, 3],
-        };
+        } as const;
```

Comment:
**question:** What is `as const` for? Why is it needed if the variable is  already `const`? Is it because TS demands that both sides of the `=` are const'd?

Reply:
I'm not sure why the tool adds `as const` in so many places.  We could change that behavior.  `as const` in this context changes the type that is inferred for the object literal.  With `as const`, it's inferred as precisely `{array: [1, 2, 3]}` whereas without it, it's inferred as `{array: number[]}`.  [TS playground example](https://www.typescriptlang.org/play?#code/MYewdgzgLgBCBOBLA5osBDANgNSwVwFMYBeGAbwCgZqZ1550BPALhgG0BGAGhgCYeAzAF0uFAL4UKoSLAQo0WXJkIBBCAGFw0EuSo06DFu259BI8bQgxp0ANxA)